go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	k8s.io/client-go v0.28.3
 	k8s.io/cluster-bootstrap v0.28.3
 	k8s.io/component-base v0.28.3
-	k8s.io/klog/v2 v2.100.1
+	k8s.io/klog/v2 v2.110.1
```

Comment:
Should we really use a version which diverges from the one used by Kubernetes 1.28? (xref: https://github.com/kubernetes/kubernetes/blob/release-1.28/go.mod#L114)